docker/replicator/Dockerfile

```diff
@@ -10,151 +9,34 @@
 #
 #   docker buildx build \
 #     --platform linux/amd64,linux/arm64 \
-#     --tag mtconnect/agent:2.0.0.12_RC18 \
+#     --tag mtconnect/replicator:latest \
 #     --push \
 #     .
 #
-#   To run tests, use `--build-arg WITH_TESTS=true`
-#
 #   # Note: In this case, I would suggest to map port `5000` to `5000`. The user can always change the port according to their needs.
-#   docker run -it --rm --init --name agent -p5000:5000 \
-#     mtconnect/agent:2.0.0.12_RC18
+#   docker run -it --rm --init --name replicator -p5000:5000 -e SOURCE_URL=http://192.168.5.100:5000/ \
```

Comment:
Consider updating `SOURCE_URL`. What should be its value? :thinking:

Reply:
It depends on what the source URL is. Don't get the point?